### ISSUE TITLE
try_mmap_metadata_chunk unit test fail for i686-unknown-linux-gnu

### ISSUE THREAD

**Issue by qinsoon** [mmtk/mmtk-core#229]
The test fails with the following error with `cargo test --target i686-unknown-linux-gnu`.
```
---- util::side_metadata::global::tests::test_side_metadata_try_mmap_metadata_chunk stdout ----
thread 'util::side_metadata::global::tests::test_side_metadata_try_mmap_metadata_chunk' panicked at 'assertion failed: try_map_metadata_space(unsafe { Address::from_usize(0) }, 1,\n                       helpers::meta_bytes_per_chunk(0, 0),\n                       helpers::meta_bytes_per_chunk(0, 1))', src/util/side_metadata/global.rs:594:9
```

We are not seeing the failing test in our CI, as it was a mistake that our `ci-test.sh` is not running tests for `i686-unknown-linux-gnu`.

**Comment by javadamiri** [comment]
This issue is still open.
Should I manually close it?
I thought my PR would automatically close this issue

**Comment by qinsoon** [comment]
It seems you would need 'closes #1234' rather than 'closes issue #1234' to automatically close an issue.
